src/java/com/zimbra/qa/selenium/projects/ajax/tests/preferences/general/VerifyTimezoneView.java

```diff
@@ -167,7 +167,7 @@ public void VerifyTimezoneView_01() throws HarnessException {
 				"GMT +12:00 Fiji",
 				"GMT +12:45 Chatham",
 				"GMT +13:00 Samoa",
-				"GMT +13:00 Nuku�alofa",
+				"GMT +13:00 Nuku\u2019alofa",
```

Comment:
can you put string value for ascii code in separate variable and use that variable here?
It will make this string more readable.